src/incremental.jl

```diff
@@ -127,7 +127,7 @@ end
     To compile just a single package, see the simpler version  `compile_incremental(package::Symbol)`:
 """
 function compile_incremental(
-        toml_path::Union{String, Nothing}, precompiles::String;
```

Comment:
What is the motivation for removing the `toml_path` argument here? It's currently unused (#222), but why shouldn't we just fix that (#259) rather than removing it? I'm using this functionality.

Reply:
OK, https://github.com/JuliaLang/PackageCompiler.jl/pull/204/commits/e12e89951919df8a82a235143ea4b2a17b9e4eb5 addresses this concern by just using the project of the parent process. Thanks, @SimonDanisch.